fdbserver/RestoreUtil.h

```diff
@@ -39,6 +39,7 @@
 //#define SevFRMutationInfo SevInfo
 
 using MutationsVec = Standalone<VectorRef<MutationRef>>;
+using SubSequenceVec = Standalone<VectorRef<uint32_t>>;
```

Comment:
I think we can do better than just encode sub-version into mutations.
In the `RestoreSendVersionedMutationsRequest`, can we have a `VersionMutations` that has the following field:
version;
Standalone<VectorRef<MutationRef, uint32_t>>, where the first is mutation, the second is the sub-version.

This will allow us to put multiple versions of mutations in one message sent from loader to applier. This can help improve performance.

Reply:
Of course, this is after the PR is correctness clean.

Reply:
That's a good idea. I'd prefer to do it as a separate/incremental PR after this one is merged.